ingest.go

```diff
@@ -1722,8 +1722,7 @@ func (d *DB) excise(
 			leftFile.ValidateVirtual(m)
 			d.checkVirtualBounds(leftFile)
 			ve.NewFiles = append(ve.NewFiles, newFileEntry{Level: level, Meta: leftFile})
-			ve.CreatedBackingTables = append(ve.CreatedBackingTables, leftFile.FileBacking)
-			backingTableCreated = true
+			needsBacking = true
 			numCreatedFiles++
 		}
 	}
```

Comment:
We early-return if there's no file to the right of the excise span below. So we'll need to copy the `if needsBacking` case below the return down in line 1732.

Reply:
Added a test for this case, and fixed it.